src/Sarif.UnitTests/Core/FileDataTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Security.AccessControl;
+using System.Text;
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public class FileDataTests
+    {
+        [Fact]
+        public void FileData_Create_NullUri()
+        {
+            Action fileDataCreation = () => { FileData.Create(null, LoggingOptions.None); };
```

Comment:
>fileDataCreation [](start = 19, length = 16)

Nothing really wrong with this. I tend to just always use the name `action` in this context.

Reply:
like it

---
In reply to: [136156396](https://github.com/Microsoft/sarif-sdk/pull/612#discussion_r136156396) [](ancestors = 136156396)